app/assets/javascripts/course/assessment/submission/submission.js

```diff
@@ -152,10 +152,32 @@
     return multiplier;
   }
 
+  function initializeAnswerTabs() {
+    $('.answers-tab a').click(function (e) {
```

Comment:
@weiqingtoh Sorry I didn't catch this earlier, these listeners need to be scoped to the submission page, not that it's global.